### DISCUSSION TITLE
请问page-loading的工作原理是什么呢，没看见在哪有相关代码呢，谢谢！

### DISCUSSION THREAD

**Discussion by laiwj** [vbenjs/vue-vben-admin#466] [Q&A]
<div class="app-loading">
        <div class="app-loading-wrap">
          <img src="/resource/img/logo.png" class="app-loading-logo" alt="Logo" />
          <div class="app-loading-dots">
            <span class="dot dot-spin"><i></i><i></i><i></i><i></i></span>
          </div>
          <div class="app-loading-title"><%= title %></div>
        </div>
      </div>
    </div>

**Comment by M69W** [comment]
index.html

**Reply by laiwj** [reply]
主要没懂它消失的机制

**Reply by M69W** [reply]
搜索出现的关键字

**Reply by 2yllll** [reply]
首先你要看mount的机制的源码，然后看一下render的机制的源码，mount后会新渲染内容替换#app的内容。
<%= title %>呢，看vite-plugin-html,了解了吧
